<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WLogDataEvery</class>
 <widget class="QDialog" name="WLogDataEvery">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>416</width>
    <height>182</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Log data period configuration</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#WLogDataEvery {
 background-color:white;
 border-radius: 5px;
}
#label{
 color: #3E4072;
 font-weight: 800;
}
#buttonBox{
 background-color: #3E4072;
}
#label_Title{
 color: #8F8F8F;
 font-weight: 600;
}
QSpinBox,
QDoubleSpinBox {
 background-color:#DFDFDF;
 border-radius: 3px;
}
#pushButton_Confirm,
#pushButton_Cancel {
 border: none;
 border-radius: 8px;
}
#pushButton_Confirm {
 color: white;
 background-color: #3E4072;
}
#pushButton_Confirm:hover {
 color: white;
 background-color: #595D89;
}
#pushButton_Cancel {
 background-color: #D8D8D8;
}
#pushButton_Cancel:hover {
 background-color: #ACACAC;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_Title">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <fontweight>DemiBold</fontweight>
      </font>
     </property>
     <property name="text">
      <string>Setup logging rate</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="4">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Secs</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Mins</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Log data every:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Hours</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QSpinBox" name="spinBox_Mins">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>28</height>
         </size>
        </property>
        <property name="maximum">
         <number>59</number>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="spinBox_Hours">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>28</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>28</height>
         </size>
        </property>
        <property name="maximum">
         <number>120</number>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QDoubleSpinBox" name="spinBox_Secs">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>28</height>
         </size>
        </property>
        <property name="decimals">
         <number>1</number>
        </property>
        <property name="maximum">
         <double>59.899999999999999</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QWidget" name="widget_3" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="pushButton_Confirm">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>120</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Confirm</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_Cancel">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>120</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_Confirm</sender>
   <signal>clicked()</signal>
   <receiver>WLogDataEvery</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>154</x>
     <y>139</y>
    </hint>
    <hint type="destinationlabel">
     <x>147</x>
     <y>275</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_Cancel</sender>
   <signal>clicked()</signal>
   <receiver>WLogDataEvery</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>330</x>
     <y>158</y>
    </hint>
    <hint type="destinationlabel">
     <x>330</x>
     <y>223</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
